gen.py

```diff
@@ -18,33 +18,35 @@
 from subprocess import PIPE, Popen, call
 try:
     from gi import require_version
-    require_version('Rsvg', '2.0')
+    require_version("Rsvg", "2.0")
```

Comment:
Switched `'` for `"` for consistency throughout. The only exceptions are strings for `app_filer` because that uses `"` characters already as part of its content.